pkg/reconciler/inmemorychannel/dispatcher/inmemorychannel.go

```diff
@@ -70,21 +77,45 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, imc *v1beta1.InMemoryCha
 }
 
 // newConfigFromInMemoryChannels creates a new Config from the list of inmemory channels.
-func (r *Reconciler) newConfigFromInMemoryChannels(channels []*v1beta1.InMemoryChannel) *multichannelfanout.Config {
+func (r *Reconciler) newConfigFromInMemoryChannels(channels []*v1beta1.InMemoryChannel) (*multichannelfanout.Config, error) {
 	cc := make([]multichannelfanout.ChannelConfig, 0)
 	for _, c := range channels {
+
+		subs := make([]fanout.Subscription, len(c.Spec.Subscribers))
```

Comment:
Could we add UT for this, looks like this path gets no love from UT?

Reply:
Done: https://github.com/knative/eventing/pull/2932/commits/030c6c62f92096b90631807cbd5d92773870c806